script/script.js

```diff
@@ -13,3 +13,232 @@ function eachClick() {
 innerButtons.forEach((item) => {
   item.addEventListener('click', eachClick);
 });
+
+const projectInfo = {
+  title: 'Keeping track of hundreds of components',
+  description: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  technologies: ['Ruby on rails', 'css', 'JavaScript'],
+  image: '../images/project-image.svg',
+  live: 'https://yothu.github.io/my-portfolio/',
+  source: 'https://github.com/Yothu/my-portfolio',
+};
+
+const projectInfoArray = [
+  projectInfo.title,
+  projectInfo.description,
+  projectInfo.image,
+  projectInfo.technologies,
+  projectInfo.live,
+  projectInfo.source,
+];
```

Comment:
**Kindly work on these required changes:**
(Use objects to store and access data.)

## In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects.**

- For each project you need to store the following pieces of data in a JavaScript object, at a minimum: name, description, featured image, technologies, link to live version, link to source.
- You need to update the main page so that the projects section is created dynamically using the information stored in that JavaScript object. Remember, all of the HTML in that section is created when the page loads.
- After that, you could implement the popup window.


_I can recommend you, just store your projects in an array and then iterate each element_.